src/httpd_jsonapi.c

```diff
@@ -1652,24 +1668,51 @@ jsonapi_reply_queue_tracks_add(struct httpd_request *hreq)
     }
 
   uris = strdup(param);
-  uri = strtok(uris, ",");
-
-  do
+  ptr = uris;
+  while (!done)
     {
+      if (addnext)
```

Comment:
lines 1674-1700 replace the original `strtok()` which allows this function to accept multiple URIs on a single call;  on the webui I thought calls to this would only provide one URI.

To keep the bulk of the code clean, we determine what the URI in this block;  unfortunately, if we have a multiple URIs (say, `a`,`b`,`c`) requested on a `play next` request, we have to add the requests backwards as we are asking the db to insert the URI after the current pos in the play Q.  ie

Current q = `x`, request to add `a, b, c`
Adding `c`, q= `x, c`.  Adding `b`, q = `x, b, c`.  Adding `a`, q =` x, a, b, c` leaving q in the order as expected.